content/index.html

```diff
@@ -22,59 +23,77 @@ <h1>Infrastructure for container projects.</h1>
 
 <div class="p-strip--light">
   <div class="row">
-    <div class="col-8">
+    <div class="col-12">
       <h2>Active projects</h2>
     </div>
   </div>
+
+  <div class="row">
+    <div class="p-card col-12">
+      <h3>LXD</h3>
+      <div class="row">
+        <div class="col-8">
+          <p>LXD is a next generation system container and virtual machine
+            manager. It offers a unified user experience around full Linux
+            systems running inside containers or virtual machines.</p>
+          
+          <p>LXD is image based and provides images for a wide number of
+            Linux distributions. It provides flexibility and scalability
+            for various use cases, with support for different storage
+            backends and network types and the option to install on
+            hardware ranging from an individual laptop or cloud instance
+            to a full server rack.</p>
+          
+          <p>When using LXD, you can manage your containers with a single
```

Comment:
How about: "When using LXD, you can managed your instances (containers and VMs) with a single..." - so as not to indicate that only containers can be managed with a single command line tool.